Unix/tests/cli/test_cli.cpp

```diff
@@ -537,7 +537,7 @@ static int StopServerSudo()
 
 cleanup:
     // To allow pid file to be deleted
-    Sleep_Milliseconds(200);
+    Sleep_Milliseconds(500);
```

Comment:
My point is it's timing sensitive. That makes it inherently risky. I don't want tests that work 90% of the time or 95% of the time, I want tests that work 100% of the time. AND: I want tests that don't take longer than they need to. So perhaps check if the PID is gone every 100 milliseconds up to 10 times or something.